### ISSUE TITLE
NFC ring troubles with alarm clock ad google Hangouts

### ISSUE THREAD

**Issue by tsener** [mclear/Android_NFC_Ring_Unlock#109]
Hi guys!
 I am pretty satisfied with my NFC ring -
I am using it all the time to unlock my phone
- with the pin, it's a very cool two-factor auth :)

However, I've stumbled upon two issues:
1. When somebody calls me via Google Hangouts (formerly Google talk)
   usually a video call- I am unable neither to unlock the phone for some reason.
   It acts like it does not detect the NFC - and I reproduce it every time
2. Same goes with my morning alarm - it's the vanilla Android 4.4.4 alarm app
   I am using; I cannot unlock, thus can't stop the alarm sound, so I have to restart the phone..

Would be very, very happy to see those fixed :)
The phone I am using is a 8GB version of LG Nexus 4, nothing special, no root no nothing. I don't have the second additional app installed, just the unlock app. Hope this helps.

Cheers and keep being awesome!

**Comment by tsener** [comment]
Ant update here ? : )

**Comment by JohnMcLear** [comment]
Not yet, we're still on the fence about how hard to push with these requests as we're not sure which way Android L is going to go as far as personal unlock is concerned.

**Comment by r2DoesInc** [comment]
It sounds as if you have the "Use with no lockscreen" option selected. This could(?) cause issues like this as it messes with the "layers" that the apps are displayed on.

**Comment by tsener** [comment]
Nope, the checkbox for  "Use with no lockscreen" is not selected.

**Comment by r2DoesInc** [comment]
The issues is that there are a million difference "Alarm" apps, so there is no way to detect an "alarm" coming in. We could do something about hangouts, but because they dont use the actual call action, its not currently accounted for in the code. We can manually look for that app, but that is something i highly recommend against.

**Comment by JohnMcLear** [comment]
Tasker in, you could use tasker to handle the event and fire an unlock event no?

**Comment by JohnMcLear** [comment]
@r2DoesInc just a note on this he does say it's the vanilla unlock method..
